client/order/index.js

```diff
@@ -129,102 +134,188 @@ jQuery( function ( $ ) {
+	let urgency = 'warning';
+	let actions;
 
-	// If the dispute is due in the past, we don't want to show the notice.
-	if ( now.isAfter( dueBy ) ) {
-		return;
-	}
+	// Refunds are only allowed if the dispute is an inquiry or if it's won.
+	const isRefundable =
+		isInquiry( dispute ) || [ 'won' ].includes( dispute.status );
+	const shouldDisableRefund = ! isRefundable;
+	let disableRefund = false;
 
-	const titleStrings = {
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		dispute_default: __(
-			// eslint-disable-next-line max-len
-			'This order has been disputed in the amount of %1$s. The customer provided the following reason: %2$s. Please respond to this dispute before %3$s.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		inquiry_default: __(
-			// eslint-disable-next-line max-len
-			'The card network involved in this order has opened an inquiry into the transaction with the following reason: %2$s. Please respond to this inquiry before %3$s, just like you would for a formal dispute.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		dispute_urgent: __(
-			'Please resolve the dispute on this order for %1$s labeled "%2$s" by %3$s.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		inquiry_urgent: __(
-			'Please resolve the inquiry on this order for %1$s labeled "%2$s" by %3$s.',
-			'woocommerce-payments'
-		),
-	};
-	const amountFormatted = formatExplicitCurrency(
-		dispute.amount,
-		dispute.currency
-	);
+	let refundDisabledNotice = '';
+	if ( shouldDisableRefund ) {
+		const refundButton = document.querySelector( 'button.refund-items' );
+		if ( refundButton ) {
+			disableRefund = true;
 
-	let urgency = 'warning';
-	let buttonLabel = __( 'Respond now', 'woocommerce-payments' );
-	let suffix = '';
-
-	let titleText = isInquiry( dispute )
-		? titleStrings.inquiry_default
-		: titleStrings.dispute_default;
-
-	// If the dispute is due within 7 days, use different wording.
-	if ( countdownDays < 7 ) {
-		titleText = isInquiry( dispute )
-			? titleStrings.inquiry_urgent
-			: titleStrings.dispute_urgent;
-
-		suffix = sprintf(
-			// Translators: %s is the number of days left to respond to the dispute.
-			_n(
-				'(%s day left)',
-				'(%s days left)',
-				countdownDays,
+			// Disable the refund button.
+			refundButton.disabled = true;
+
+			const disputeDetailsLink = getDetailsURL( dispute.id, 'disputes' );
+			// TODO: finalize text copy after we come up with the final decision.
+			refundDisabledNotice = createInterpolateElement(
+				__(
+					'Refunds and order editing are disabled because this order was disputed. <a>View dispute</a>',
```

Comment:
It's not clear what dispute status this text will render for, since there is an `if` statement for `lost` disputes further down.